templates/slides.tmpl

```diff
@@ -105,8 +105,25 @@
         languages: ['bash', 'markdown', 'go', 'java']
       })
 
+      var reLanguageTag = /^\s*language:(\w+)\n?([\s\S]*)/m
+
       document.querySelectorAll('.code pre').forEach( el => {
```

Comment:
I think I'd invert the logic of this a bit to have less nesting in the if-statements, but really that's a matter of taste. `go` is probably leaving its marks on me. Feel free to ignore (code untest):

```js
document.querySelectorAll('.code pre').forEach( el => {
  var match = reLanguageTag.exec(el.innerText)
  if (!match) {
    hljs.highlightBlock(el)
    return
  }
  var language = match[1]
  if (language === 'none') {
    return
  }
  el.classList.add(language)
  el.innerText = match[2].trim()
  hljs.highlightBlock(el)
})
```

Reply:
Thanks Julia, the thought of an early out crossed my frazzled mind last night but I must have blanked

(I blame Christian Bale for being an astonishing Dick Cheney)

Will update ASAP

Reply:
I skipped your second early-out (`'none'`), as the intent was for a `.none` class to be added when syntax wasn't applied (allowing specialised overrides if deemed necessary)

Otherwise - all done!

Reply:
To add to the bikeshedding - when writing code snippets please use language tags e.g. ` ```js ` for syntax highlighting (how meta).

```js
document.querySelectorAll('.code pre').forEach( el => {
  var match = reLanguageTag(el.innerText)
  if (match) {
    var languageTag = match[1]
    el.innerText = match[2].trim()
    el.classList.add(languageTag)
    if (languageTag === 'none') {
      return
    }
  }
  hljs.highlightBlock(el)
})
```

But yes - clearest of them all. 
And yes, let's not bother with re-work.

Reply:
Thanks, that was niggling at my cloudy mind

Would that I had a moment to sneak it into the PR - hide the crimes

Reply:
ok, i added a language tag. thanks @juliaogris - that was something I once knew (vaguely) but had long since forgotten.